apps/pie-storybook/stories/testing/pie-radio-group.test.stories.ts

```diff
@@ -48,7 +48,7 @@ export default radioGroupStoryMeta;
 const KeyboardNavigationTemplate = () => html`
     <h2>Radio group 1</h2>
     <p><pie-button size="small-productive" data-test-id="btn-1">Button 1</pie-button></p>
-    <pie-radio-group data-test-id="radio-group-1">
+    <pie-radio-group name="radio-group-1" data-test-id="radio-group-1">
```

Comment:
Can we also add the name to radio-group-2 for consistency?

Reply:
@siggerzz this is literally just for testing purposes and I'd prefer to keep this markup minimal and only add what is used for testing. So I see no reason to add a name property to the second radio group at this time

Reply:
While I appreciate the idea @siggerzz , I think having a side-by-side comparison of how the radio-group behaves with and without a name probably offers a more tangible benefit than consistency. Also I agree with @jamieomaguire's point about the test markup being lean, so we know that everything is necessary for the tests we have